miraheze/swift/fix_container_permissions.py

```
@@ -1,15 +1,37 @@
 import re
 import subprocess
-import argparse
 
 
-def fix_container_perms(wiki: str) -> None:
-    out = subprocess.run(['sudo', '-u', 'www-data', 'php', '/srv/mediawiki/1.43/maintenance/run.php', 'CreateWiki:SetContainersAccess', '--wiki', wiki], capture_output=True, text=True)
-    matches = re.findall(r"Making sure 'mwstore:\/\/miraheze-swift\/([^']+)' [^\n]+\.failed\.", out.stdout)
-    for match in matches:
-        subprocess.run(['swift', 'post', '--read-acl', 'mw:media', '--write-acl', 'mw:media', f'miraheze-{wiki}-{match}'], check=True)
+def run_mwscript_setcontainersaccess(wiki: str, check: bool) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)